examples/falcon/config.json

```diff
@@ -53,14 +53,7 @@
         }
     },
     "engine": {
-        "search_strategy": {
-            "execution_order": "pass-by-pass",
-            "search_algorithm": "tpe",
-            "search_algorithm_config": {
-                "num_samples": 3,
-                "seed": 0
-            }
-        },
+        "search_strategy": false,
```

Comment:
BTW, should the hf config of the input model be updated to use `trust_remote_code=True`?

Reply:
Falcon model doesn't need this but Yi model needs. I'm working on yi model.